src/shipit_code_coverage/tests/test_artifacts.py

```diff
@@ -59,3 +61,27 @@ def add_dir(files):
 
     with pytest.raises(Exception, message='suite and chunk can\'t both have a value'):
         a.get(chunk='xpcshell-7', suite='mochitest')
+
+
+#I'm using the LINUX_TASK_ID as taskID here, but the problem is that,
+#when I do taskcluster.get_task_artifacts, none of the names has code-coverage-grcov.zip
+#or code-coverage-jsvm.zip as substring, so artifact.download() never calls
+#taskcluster.download_artifact(). Is there a specific TEST_TASK I should use?
+TEST_TASK = {'task' : {'metadata' : {'name' : 'test-linux64-ccov/opt/test/code-coverage-grcov.zip'}},
+             'status' : { 'taskId' : 'MCIO1RWTRu2GhiE7_jILBw' }
+}
+
+
+@responses.activate
+def test_download(LINUX_TASK_ID, LINUX_TASK_ARTIFACTS, FAKE_ARTIFACTS_DIR):
+    #I want to use this to write the paths of files downloaded.
+    def add_dir(files):
+        return set([os.path.join(FAKE_ARTIFACTS_DIR, f) for f in files])
+
+    a = ArtifactsHandler([], [], parent_dir=FAKE_ARTIFACTS_DIR)
+
+    #mock the taskcluster.get_task_artifacts()
+    responses.add(responses.GET, 'https://queue.taskcluster.net/v1/task/{}/artifacts'.format(LINUX_TASK_ID), json=LINUX_TASK_ARTIFACTS, status=200)
```

Comment:
I suggest actually mocking here, rather than adding fake responses.
You can use https://pypi.python.org/pypi/pytest-mock.